components/Article/PayNote.js

```diff
@@ -235,12 +235,24 @@ const generateKey = (note, index) => {
   return { ...note, key: `custom-${index}` }
 }
 
+const isPermissibleIOSCta = cta => !cta || cta === 'trialForm'
```

Comment:
wouldn't `cta !== 'button'` be less convoluted?

Reply:
Would be shorter but generally I prefer declaring what is allowed: no cta and `trialForm`. IMO easier to understand what the policy is.